src/Event/EventManagerInterface.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @since         3.6.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Event;
+
+interface EventManagerInterface
```

Comment:
@robertpustulka have you seen this one? https://github.com/php-fig/fig-standards/blob/master/proposed/event-manager.md Scroll down to the interface for the manager.

Reply:
Yes I have. It’s in proposed for almost two years, I’m not sure if anyone’s interested in working on it.